src/components/ChallengePane/ChallengePane.jsx

```diff
@@ -43,11 +43,20 @@ const ShowArchivedToggleInternal = (props) => {
         }}
       />
       <div className="mr-text-sm mr-mx-1"><FormattedMessage {...messages.showArchivedLabel} /></div>
+      <input
+        type="checkbox"
+        className="mr-checkbox-toggle mr-mr-1 mr-mb-6 mr-ml-4"
+        checked={props.showingGlobal}
```

Comment:
Is the intent for global challenges to be shown by default, or hidden?

Reply:
The intent is it to be hidden... The condition you pointed out below was reversing this